app/views/layouts/_sidebar.html.erb

```diff
@@ -5,7 +5,10 @@
     <% if can? :manage, User %>
       <%= link_to 'Users', users_path, class: 'list-group-item list-group-item-action bg-light' %>
     <% end  %>
-    <%= link_to 'Sets', admin_sets_path, class: 'list-group-item list-group-item-action bg-light' %>
+    <% if can? :manage,  AdminSet %>
+      <%= link_to 'Sets', admin_sets_path, class: 'list-group-item list-group-item-action bg-light' %>
+    <% end  %>
+    <%#= link_to 'Sets', admin_sets_path, class: 'list-group-item list-group-item-action bg-light' %>
```

Comment:
All looks nice. Maybe this comment out line #11  can be deleted. This line was duplicated.

Reply:
Great catch,  I deleted the line.